.github/scripts/plugin-health-check.sh

```
@@ -0,0 +1,97 @@
+# Get all plugin repos
+repos=$(gh api "orgs/$ORG/repos?per_page=100&sort=updated" --jq '.[].name')
+
+failed_repos=()
+
+for repo in $repos; do
+  echo "Checking $repo..."
+
+  # Get recent workflow runs (only failed and success to count consecutively)
+  runs=$(gh api "repos/$ORG/$repo/actions/runs?per_page=$RECENT_RUNS&status=failure" --jq '.workflow_runs[].id' 2>/dev/null || true)
+
+  if [ -z "$runs" ]; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)